PizzaDAO/pizza-oven-py
# Add IPFS node

add ipfs node to docker compose and validate data can be posted to it.

To test:

`make docker-dev` and run the create order endpoint from postman.  at a certain point you'll get a real ipfs hash back, browse to it in a broweser and verify it is there
